libs/ledger-live-common/src/deviceSDK/commands/quitApp.ts

```diff
@@ -0,0 +1,16 @@
+import { Observable } from "rxjs";
+import Transport from "@ledgerhq/hw-transport";
+
+
+export function quitApp(
+  transport: Transport
+): Observable<void> {
+  return new Observable((observer) => {
```

Comment:
Should it implement the unresponsive timeout ? `transport.setExchangeUnresponsiveTimeout(1000)` etc. like `getVersion` for ex ?

Reply:
I documented the command on its behavior with locked devices - what i was proposing is not needed